python/cudf_polars/cudf_polars/dsl/translate.py

```diff
@@ -690,6 +690,18 @@ def _(node: pl_expr.SortBy, translator: Translator, dtype: plc.DataType) -> expr
     )
 
 
+@_translate_expr.register
+def _(node: pl_expr.Slice, translator: Translator, dtype: plc.DataType) -> expr.Expr:
+    offset = translator.translate_expr(n=node.offset)
+    length = translator.translate_expr(n=node.length)
+    return expr.Slice(
+        dtype,
+        offset.value.as_py(),  # type: ignore[attr-defined]
```

Comment:
question: Is there a way to _not_ ignore the mypy check here? Basically we annotate `translator.translate_expr` to return an `expr.Expr` which doesn't have a "value" attribute. But offset does have a "value" attr because it is a `expr.Literal`.

Reply:
Ah okay thanks! I'll add those asserts